application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -61,20 +57,18 @@ public void runRoutine(JDA jda) {
                 .where(PENDING_REMINDERS.REMIND_AT.lessOrEqual(now))
                 .stream()
                 .forEach(pendingReminder -> {
-                    sendReminder(jda, pendingReminder.getId(), pendingReminder.getChannelId(),
-                            pendingReminder.getAuthorId(), pendingReminder.getContent(),
-                            pendingReminder.getCreatedAt(), pendingReminder.getGuildId());
+                    Reminder reminder = new Reminder(pendingReminder.getId(),pendingReminder.getCreatedAt(),
+                            pendingReminder.getGuildId(),pendingReminder.getChannelId(),pendingReminder.getAuthorId(),
+                            pendingReminder.getRemindAt(),pendingReminder.getContent(),pendingReminder.getFailureAttempts());
```

Comment:
u can improve the code if u give ur `Reminder` record a method:
```java
public static Reminder from(PendingReminder pendingReminder) {
  return new Reminder(...);
}
```
then u can write `sendReminder(jda, Reminder.from(pendingReminder));` here

Reply:
can i actually do that? couldn't find anything related to from keyword in `records`

Reply:
records are just classes, u can give them methods:
```java
record Person(String name, int age) {
  public static Person from(String line) {
    ...
  }
}
```

Reply:
![image](https://user-images.githubusercontent.com/61616007/220651359-6dbd943b-d1b3-46b6-a154-07b4a7eceb5e.png)
 is this really better? kinda makes `getters` longer than how it used to be
for ex: earlier i could access any field using `reminder.field`, but now it has to be `reminder.pendingReminders().field()`

Reply:
no. u misunderstood me here. i meant like:

```java
record Reminder(String name, int age) {
  public static Reminder from(PendingReminder other) {
    return new Reminder(other.getName(), other.getAge())
  }
}
```
the `Reminder` was not supposed to wrap the `PendingReminder`, but to extract all its content into its own fields.

Reply:
should i hold onto this one for now ? taz mentioned to pass on pendingReminder itself so

Reply:
if its possible to use `pendingReminder` itself, then i would prefer that and delete your `Reminder` record